app/components/rate-goal/animations/cactus.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import GSAP from 'react-gsap-enhancer';
+import { TweenMax, TimelineMax, } from 'gsap';
+
+const createAnimation = ({target, options, }) => {
+
+  const bodyFrames = [
+    { bottom: '50px', height: '0px', },
+    { bottom: '54px', height: '50px', },
+    { bottom: '66px', height: '50px', },
+    { bottom: '70px', height: '60px', },
+    { bottom: '75px', height: '65px', },
+    { bottom: '75px', height: '70px', },
+    { bottom: '80px', height: '70px', },
+    { bottom: '85px', height: '70px', },
+    { bottom: '90px', height: '72px', },
+    { bottom: '95px', height: '77px', },
+    { bottom: '97px', height: '81px', },
+  ];
+
+  const armFrames = [
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'hidden' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+    { visibility: 'visible' },
+  ];
+
+  const vase = target.find({ id: 'vase' });
+  const cactus3 = target.find({ id: 'cactus3' });
+  const arms = target.find({ id: 'arms' });
+
+  let t1 = new TimelineMax();
+
+    const sequence = options.score > options.previousScore
+      ? bodyFrames.slice(options.previousScore, options.score + 1)
+      : bodyFrames.slice(options.score, options.previousScore + 1).reverse();
+
+      const armSequence = options.score > options.previousScore
+        ? armFrames.slice(options.previousScore, options.score + 1)
+        : armFrames.slice(options.score, options.previousScore + 1).reverse();
+
+    sequence.forEach((frame, index) => {
```

Comment:
you can use zip
zip2(s1, s2).forEach(([s1Elem, s2Elem]) => {})

But you do not have to do this if you do not want to
```js
const zip2 = (xx, yy) => {
  const xl == xx.length
  const yl == yy.length
  const l = xl < yl ? xl : yl
  const out = []
  for(let i =0; i < l; i++) {
    out[i] = [xx[i], yy[i]
  }
return out
}
```